src/bump/performBump.ts

```diff
@@ -52,6 +54,33 @@ export async function updatePackageLock(cwd: string): Promise<void> {
     if (!res.success) {
       console.warn('Updating package-lock.json failed. Continuing...');
     }
+  } else if (root && fs.existsSync(path.join(root, 'pnpm-lock.yaml'))) {
+    console.log('Updating pnpm-lock.yaml after bumping packages');
+    const res = await pnpm(['install', '--lockfile-only', '--ignore-scripts'], { stdio: 'inherit' });
+    if (!res.success) {
+      console.warn('Updating pnpm-lock.yaml failed. Continuing...');
+    }
+  } else if (root && fs.existsSync(path.join(root, 'yarn.lock'))) {
+    console.log('Updating yarn.lock after bumping packages');
```

Comment:
The lock file for yarn v1 doesn't reflect local packages, so this step should be skipped for yarn v1.